pulser-core/pulser/devices/_device_datacls.py

```diff
@@ -91,6 +94,9 @@ class BaseDevice(ABC):
         min_layout_traps: The minimum number of traps a layout can have.
         max_layout_traps: An optional value for the maximum number of traps a
             layout can have.
+        min_layout_filling: The smallest fraction of a layout that must be
+            filled with atoms. Only enforced for layouts with more traps than
+            'min_layout_traps'.
```

Comment:
What does the last sentence mean. It shouldn't be possible to have layouts with less traps then `'min_layout_traps'`?

Reply:
I agree with you, I find the last sentence a bit weird. 
```suggestion
            filled with atoms. Checked for layouts validating the condition of having more traps than 'min_layout_traps'.
```

Reply:
Indeed, but they can have exactly `min_layout_traps`, in which case we don't enforce `min_layout_filling` to allow for arbitrarily small registers

Reply:
Ah, I didn't have this edge case by reading the sentence...
```suggestion
            filled with atoms. Only enforced for layouts with strictly more traps than
            'min_layout_traps'.
```